src/components/RoomMainUi.tsx

```diff
@@ -1,16 +1,55 @@
-// HOST ROOM
 'use client'
 
 import { socketRoomEmitter } from '@/services/socket'
-
-import StoryPoints from '@/components/StoryPoints'
+import { useSocketListener } from '@/services/socket'
+import { useState, useEffect } from 'react'
+import StoryPointsContainer from '@/components/StoryPointsContainer'
 import CardsContainer from '@/components/CardsContainer'
 
-export default function RoomMainUi() {
+export type RoomInfoData = {
+	roomId: string
+	userName: string
+}
+
+export default function RoomMainUi({ roomId, userName }: RoomInfoData) {
+	// const [selectedStoryPoint, setSelectedStoryPoint] = useState<string>('')
+
+	// const resetStoryPointBtns = useSocketListener('reset-points')
+
+	// useEffect(() => {
+	// 	if (selectedStoryPoint === '') return
+	// 	const timeStamp = Date.now().toString()
+	// 	socketRoomEmitter(
+	// 		'story-points',
+	// 		selectedStoryPoint,
+	// 		userName,
+	// 		timeStamp,
+	// 		roomId,
+	// 	)
+	// }, [selectedStoryPoint, roomId, userName])
+
+	// useEffect(() => {
+	// 	if (!resetStoryPointBtns) return
+	// 	console.log(
+	// 		'%c>>> resetStoryPointBtns',
+	// 		'color: red',
+	// 		resetStoryPointBtns,
+	// 	)
+	// 	if (resetStoryPointBtns.message === 'reset') {
+	// 		setSelectedStoryPoint('')
+	// 	}
+	// }, [resetStoryPointBtns])
+
 	return (
-		<div className='h-full w-full flex justify-center items-start gap-10 border-0 border-stone-900 '>
-			<StoryPoints />
-			<CardsContainer />
+		<div className='h-full w-full flex flex-col justify-start items-center gap-10'>
```

Comment:
praise: The page with all the points and cards is very cute ehehe

Reply:
Thanks! It'll get better when I do the rest of the styling.